papaparse.js

```diff
@@ -415,6 +415,50 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
+	{
+		this._columnConfig = {};
+		this._isAllDynamic = false;
+		this._isBlackListOnly = true;
+
+		if (config === true)
+		{
+			this._isAllDynamic = true;
+			return;
+		}
+		else if (!config)
+		{
+			return;
+		}
+
+		if (config.__only instanceof Array)
+		{
+			this._isBlackListOnly = false;
+			config.__only.forEach(function(field)
```

Comment:
I agree that it looks better, but I think it's necessary to be backwards with the old way of configuring things. If a file happens to have columns named `'only'` or `'except'`, then it would try to cast those column types.

If we decide that we want to remove those underscores, then we should probably deprecate the old way of configuring `dynamicTyping` with a hash & remove that support in the next major release.

Thoughts?